_posts/en/newsletters/2022-08-24-newsletter.md

```diff
@@ -0,0 +1,134 @@
+  list that they've [published][rn jam] a guide to [channel jamming
+  attacks][topic channel jamming attacks] and several proposed
+  solutions.  The guide also examines how some solutions can benefit
+  protocols building on top of LN, such as swap protocols and
+  short-duration [DLCs][topic dlc].
+
+- **Updated silent payments PR:** woltx [posted][woltx sp] to the
+  Bitcoin-Dev mailing list that the PR to Bitcoin Core for [silent
+  payments][topic silent payments] has been updated.  Silent payments
+  provide an address that can be reused by different spenders without
+  creating a link between those spends that's observable onchain (though
+  the receiver needs to be careful not to weaken that privacy through
+  their subsequent actions).  The most significant change to the PR is
+  the addition of a new type of [output script descriptor][topic
+  descriptors] for silent payments.
+
+    The design of the new descriptor received quite a bit of discussion
+    on the PR.  It was noted that only allowing a single silent payment
+    descriptor per wallet would be most efficient for monitoring for new
+    transactions but that it would also create a bad experience for
+    users in many case.  A slight tweak to the silent payment design was
+    proposed to address the issue, although it also came with trade
+    offs.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Purse.io adds Lightning support:**
+  In a [recent announcement][purse ln tweet], Purse.io announced support for
+  depositing (receiving) and withdrawing (sending) using the Lightning Network.
+
+- **Proof of concept coinjoin implementation joinstr:**
+  1440000bytes developed [joinstr][joinstr github], a proof of concept coinjoin
+  implementation using the [nostr protocol][nostr github], a public key based
+  relay network with no central server.
+
+- **Coldcard firmware 5.0.6 released:**
+  Coldcard’s version 5.0.6 adds more support for [BIP85][], `OP_RETURN` scripts,
+  and multisig [descriptors][topic descriptors].
+
+- **Nunchuk adds taproot support:**
+  In the latest versions of [Nunchuk's mobile wallet][nunchuk appstore],
+  [taproot][topic taproot] (single-sig), [signet][topic signet], and enhanced
+  [PSBT][topic psbt] support were added.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BDK 0.21.0][] is the latest release of this library for building
+  wallets.
+
+- [Core Lightning 0.12.0rc2][] is a release candidate for the next major
+  version of this popular LN node implementation.
+
+- [LND 0.15.1-beta.rc1][] is a release candidate that "includes support
+  for [zero conf channels][topic zero-conf channels], scid [aliases][],
+  [and] switches to using [taproot][topic taproot] addresses everywhere
```

Comment:
Unless I'm missing something?
```suggestion
  and switches to using [taproot][topic taproot] addresses everywhere
```

Reply:
This is a quote from their release notes where elided some details and inserted the conjunction.  I put the modification in brackets to indicate it's not original (i.e., this is deliberate and not releated to markdown).